### ISSUE TITLE
Что делать, если скрипт ничего не находит

### ISSUE THREAD

**Issue by stasink** [AiratHalitov/airat-aibolit-update#3]
В чём может быть проблема? PHP 7.4 - сканирует, никаких ошибок... Ни одного срабатывания...  Ну не может такого быть, чтобы у 2 дюжин сайтов, не было ни одного хотя бы мало-мальски  подозрительного файла. Без малого 900000 файлов. Может сервер не так настроен.  Ну разрешения там всякие... Или Jail мешает. В логах ничего необычного не нашёл. Куда копать?

**Comment by AiratHalitov** [comment]
Проверяли в обычном и параноидальном режиме?

```
php ai-bolit-hoster.php --mode=1
php ai-bolit-hoster.php --mode=2
```

Как я написал в репозитории - разработчики перестали поддерживать этот скрипт. Поэтому новые уязвимости он вряд ли найдет. Только то, что было до последнего обновления. (_Если только мы сами не начнем его обновлять и поддерживать_)

При сканировании файлы должны быть доступны для скрипта. Надежнее запускать с правами рута. 

Он у меня тоже ничего не находил, но я использовал его для подстраховки и проверял всё другим сканнером для надежности. И другой сканнер тоже ничего не находил.

**Comment by stasink** [comment]
Да, в обоих режимах проверял.  А другой скрипт это который?  Если не секрет... Поддерживать вряд ли получится, потому что часть кода обфусцировано. А так бы, конечно, хотелось развития.

**Comment by AiratHalitov** [comment]
Плагин для WordPress:
Wordfence Security https://ru.wordpress.org/plugins/wordfence/

Если используете сайты на WordPress - то рекомендую его установить и настроить. 
Он так же сравнивает файлы плагинов с сайта на соответствие с файлами из репозитория WP.
И сканирует сайт периодически в фоне. 
Антивирус + файервол.

**Comment by stasink** [comment]
Спасибо Айрат попробую.

**Comment by AiratHalitov** [comment]
Оставлю в качестве справки

**Comment by a2u** [comment]
> Как я написал в репозитории - разработчики перестали поддерживать этот скрипт.

Добрый день! Скрипт продолжает развиваться, только теперь он интегрирован в [Imunify360 / ImunifyAV](https://www.imunify360.com/).
